src/container/RelayRenderer.js

```diff
@@ -342,12 +287,50 @@ class RelayRenderer extends React.Component {
     this.mounted = false;
   }
 
+  /**
+   * @private
+   */
+  _resolveContainerProps(): Object {
+    return this.containerProps = (
+      this.containerProps ||
+      {
+        ...this.props.queryConfig.params,
+        ...mapObject(
+          this.querySet,
+          query => createFragmentPointerForRoot(this.props.environment, query)
+        ),
+      }
+    );
+  }
+
+  /**
+   * @private
+   */
+  _calculateRenderArgs(): RelayRendererRenderArgs {
+    const {readyState} = this.state;
+    return readyState ?
+      {
+        done: readyState.done,
+        error: readyState.error,
+        props: readyState.ready ? this._resolveContainerProps() : null,
+        retry: this._retry.bind(this),
+        stale: readyState.stale,
+      } :
+      {
+        done: false,
+        error: null,
+        props: null,
+        retry: this._retry.bind(this),
+        stale: false,
+      };
```

Comment:
Not a problem, replaced with if/else. But it looks bit bulkier and harder to read, IMHO.